code/__DEFINES/DNA.dm

```diff
@@ -51,6 +51,7 @@
 #define GELADIKINESIS		/datum/mutation/human/geladikinesis
 #define CRYOKINESIS /datum/mutation/human/cryokinesis
 #define CLUWNEMUT   /datum/mutation/human/cluwne
+#define SUPERANTIGLOWY
```

Comment:
chłopie zdefiniuj to

Reply:
a, kurde. Jak to się? wejść na vs code i to naprawić?

Reply:
wejść, naprawić, commit + push to origin

Reply:
ja to zrobiłem na gicie, ta naprawa